chromium_src/BUILD.gn

```diff
@@ -66,6 +67,37 @@ static_library("chrome") {
     deps += [ "//ui/snapshot" ]
   }
 
+  if (enable_color_chooser) {
+    sources += [
+      "//chrome/browser/platform_util.cc",
+      "//chrome/browser/platform_util.h",
+      "//chrome/browser/ui/browser_dialogs.h",
+    ]
+
+    if (use_aura) {
+      sources += [
+        "//chrome/browser/platform_util_aura.cc",
+        "//chrome/browser/ui/views/color_chooser_aura.cc",
+        "//chrome/browser/ui/views/color_chooser_aura.h",
+      ]
+    }
+
+    if (is_mac) {
+      sources += [
+        "//chrome/browser/ui/cocoa/color_chooser_mac.h",
+        "//chrome/browser/ui/cocoa/color_chooser_mac.mm",
```

Comment:
FYI, anything in a `cocoa/` directory on with `_mac` in its name is automatically only included in `sources` if `is_mac`. See https://chromium.googlesource.com/chromium/src/+/69.0.3497.106/build/config/BUILDCONFIG.gn#398

Reply:
TIL